src/Adaptor/FontManager.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace PHPfriends\SimplePdf\Adaptor;
+
+class FontManager
+{
+    private static $instance = null;
+
+    /** @var array */
+    protected $dict = [];
+
+    /**
+     * @return FontManager
+     */
+    public static function getInstance()
+    {
+        if(null === static::$instance){
+            static::$instance = new FontManager();
+        }
+
+        return static::$instance;
+    }
+
+    /**
+     * @param string $name
+     * @return string
+     */
+    public function addFontName($name)
+    {
+        if(false === ($alias = $this->getAliasByName($name))){
+            $alias = 'F'.(count($this->dict)+1);
+            $this->dict[$alias] = $name;
+        }
+
+        return $alias;
+    }
+
+    /**
+     * @param $alias
+     * @return string
+     */
+    public function getNameByAlias($alias)
```

Comment:
but the method doesn't return the font, only the name, it's kind of dictionary system in order to use the same alias always for the same name of the font